imagehash/__init__.py

```diff
@@ -19,7 +19,7 @@
 >>> for r in range(1, 30, 5):
 ...     rothash = imagehash.average_hash(Image.open('test.png').rotate(r))
 ...     print('Rotation by %d: %d Hamming difference' % (r, hash - rothash))
-... 
+...
```

Comment:
Okay, I tried.  I even added a space through vi right before I committed the change.